config/vufind/ProQuestFSG.ini

```diff
@@ -0,0 +1,162 @@
+
+; This controls whether results are loaded with JavaScript when paging or changing
+; settings. Loading results this way improves performance and accessibility, and is
+; enabled by default. Loading will automatically fall back to non-JS mode if
+; JavaScript is not supported by the browser.
+load_results_with_js = true
+
+; This setting can be used to configure pagination control on top of results.
+; Possible options are:
+; empty string or false  No top pagination (default)
+; simple                 Simple next/prev button pagination
+; full                   Full pagination alike to the one at the bottom of results
+;top_paginator = simple
+
+; The following two sections can be used to associate specific recommendations
+; modules with specific search types defined in the [Basic_Searches] section
+; below.  For all the details on how these sections work, see the comments above
+; the equivalent sections of searches.ini.  Recommendations work the same in
+; ProQuestFSG as they do in the regular Search module.
+;[SideRecommendations]
+; No search-specific settings by default -- add your own here.
+;[TopRecommendations]
+; No search-specific settings by default -- add your own here.
+
+; The order of display is as shown below
+; The name of the index field is on the left
+; The display name of the field is on the right
+[Facets]
+Databases  = "Source"
+
+; This section is reserved for special boolean facets.  These are displayed
+; as checkboxes.  If the box is checked, the filter on the left side of the
+; equal sign is applied.  If the box is not checked, the filter is not applied.
+; The value on the right side of the equal sign is the text to display to the
+; user .  It will be run through the translation code, so be sure to update the
+; language files appropriately.
+[CheckboxFacets]
+limitFullText = "fulltext_limit"
+limitScholarlyJournals = "scholarly_limit"
+limitPeerReviewed = "peer_reviewed_limit"
+
+; Facet display settings
+[Results_Settings]
+; By default, the side facets will only show 6 facets and then the "show more"
+; button. This can get configured with the showMore settings.
+; You can use the * to set a new default setting.
+showMore[*] = 6
+; Or you can set a facet specific value by using the facet name as index.
+;showMore['rtype'] = 10
+; Do we want any facets to be collapsed by default?
+;collapsedFacets = *
+
+; This section shows which search types will display in the basic search box at
+; the top of ProQuest pages.  The name of each setting below corresponds with an
+; index defined in the ProQuestFSG API.  The value of each setting is the text to
+; display on screen.  All on-screen text will be run through the translator, so
+; be sure to update language files if necessary.  The order of these settings
+; will be maintained in the drop-down list in the UI.
+[Basic_Searches]
+cql.serverChoice    = "All Fields"
+title               = Title
+author              = Author
+subject             = Subject
```

Comment:
@sturkel89 noticed one last problem: if you go to a record and click on a subject in the core metadata, the resulting search results have the "All Fields" option selected in the drop-down, even though a Subject search is being performed. This appears to be due to the default link-subject.phtml template using "Subject" and this configuration using "subject." If the API is case-sensitive, you'll probably want to create a custom link-subject.phtml template (just as you have already created link-author.phtml). If it is not case-sensitive, maybe it's worth considering changing the names here so things match up using default templates.

Reply:
Good catch @sturkel89  -- this is fixed.  That part of the API actually appears to be case-insensitive, but the documentation has everything lower case so I'd rather just match it in case some other part of it turns out to care.